src/TrackerCouncil.Smz3.UI/Views/AutoTrackingHelpWindow.axaml

```diff
@@ -44,8 +44,48 @@
         • <controls:LinkControl Click="BizHawkButton_OnClick">BizHawk</controls:LinkControl> (BSNES Core Only)
       </TextBlock>
 
+      <TextBlock TextWrapping="Wrap" Margin="15,5,0,0">
+        An alternate Lua Script mode for compatibility with the EmoTracker Lua script is also available if you use that for other randomizers.
```

Comment:
Is all the information from this help window accessible through the setup window? I imagine someone who's relatively new to all this might need help figuring out which is the right option, especially considering some of the emulators we list appear in multiple options.

Reply:
Yeah, the more info button opens this up.